mucommander-viewer-text/src/main/java/com/mucommander/viewer/text/TextEditorImpl.java

```diff
@@ -232,23 +234,42 @@ void findPrevious() {
     void convertSpacesToTabs() { textArea.convertSpacesToTabs(); }
 
     private void doSearch(boolean forward) {
+        doSearchAndReplace(forward, null);
+    }
+
+    /**
+     * Does Search or Replace if replaceWith is not null.
+     *
+     * @param forward search/replace forward
```

Comment:
but there's no replace-forward, right? so this need to change to something like
```suggestion
     * @param forward search forward if 'true' or backward otherwise., ignored when replaceWith is not null
```